src/components/PeoplePage.tsx

```diff
@@ -117,6 +118,16 @@ export const PeoplePage = () => {
     }
   };
 
+  const getAllCenturies = (peopleArray: Person[]) => {
+    const centuries: string[] = [];
+
+    peopleArray.map((person) => {
+      return centuries.push(String(Math.ceil(person.born / 100)));
+    });
+
+    return [...new Set(centuries)];
```

Comment:
mm, i'd use `.forEach` instead of `.map` :)
just from the method purpose perspective:
- `.map` returns a new array,
- while `.forEach` is intended for iteration

```suggestion
peopleArray.forEach(({ born }) => {
  const bornCentury = Math.ceil(born / 100);
  if (!centuries.includes(bornCentury) {
    centuries.push(bornCentury)
  }

  return centuries
}
```